integration/terraform/ec2/linux/main.tf

```diff
@@ -24,7 +24,8 @@ resource "aws_instance" "integration-test" {
       "export AWS_REGION=${var.region}",
       "echo run tests with the tag integration, one at a time, and verbose",
       "cd ~/amazon-cloudwatch-agent",
-      "go test ./integration/test/... -p 1 -v --tags=${var.tag}"
+      "echo run sanity test && go test ./integration/test/sanity -p 1 -v --tags=integration",
```

Comment:
This is interesting. Question - is there any reason we wouldn't want to break out the sanity test as a separate workflow from the integration tests? 🤔

Reply:
Sanity is here to test if things are working on the box. Will tell us our agent is working. I like sanity for every test case. Just makes sense to me. We can talk about this.

Reply:
When we break out all of the integration tests by directory, does that mean we would use different hosts to run like the CA bundle tests versus the dimensions tests versus the CWL tests? If that's the case, then yeah this makes sense to me. I think when I was reading it last night, I was still in the frame of reference where the tests would be executed on the same hosts, so we only need to run sanity once, not per directory.